bin/upload-data.py

```diff
@@ -69,8 +71,14 @@ def main():
     if not os.path.isabs(folder):
         folder = os.path.join(config.service_folder, folder)
         print("Folder is relative, using", folder)
+    files += args.files
     if args.common is not None:
-        copy_everywhere(config, args.common, folder, copyOptions)
+        files.append(args.common)
```

Comment:
Only args.common file is copied to all machines, the other files are copied in round-robin amongst all machines.

Reply:
Yeah,  got it.  Will change.

Reply:
@mbudiu-vmw Done.